src/app/authentication/auth.ts

```diff
@@ -1,19 +1,20 @@
-/* tslint:disable */
 // ------------------------------------------------------------------------------
-//  Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.  See License in the project root for license information.
+//  Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.
+//  See License in the project root for license information.
 // ------------------------------------------------------------------------------
 
 import { ChangeDetectorRef } from '@angular/core';
-
 import { AppComponent } from '../app.component';
 import { IExplorerOptions, IMessage } from '../base';
 import { GraphService } from '../graph-service';
 import { PermissionScopes } from '../scopes-dialog/scopes';
 import { getParameterByName } from '../util';
+import { AuthService } from './auth.service';
 
-export function initAuth(options: IExplorerOptions, apiService: GraphService, changeDetectorRef: ChangeDetectorRef) {
-  setInterval(refreshAccessToken, 1000 * 60 * 10); // refresh access token every 10 minutes
-  hello.init({
+export function initAuth(options: IExplorerOptions, apiService: GraphService, changeDetectorRef: ChangeDetectorRef,
+                         authService: AuthService) {
+    setInterval(refreshAccessToken, 1000 * 60 * 10); // Refresh access token every 10 minutes
+    hello.init({
```

Comment:
Why do we still have references to hello.js here?  I thought this entire auth.ts was being replaced by auth.service,ts.